demo/src/main/java/hu/flowacademy/stockmarket/persistance/model/Stock2.java

```diff
@@ -0,0 +1,132 @@
+package hu.flowacademy.stockmarket.persistance.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Table(name = "stocks2")
+public class Stock2 {
```

Comment:
Ezt a nevezéktant azért gondold még át, ne legyen számozott, ha nem muszáj. Legyen inkább a szolgáltatótól származó neve pl.: `YahooStock` vagy hasonló

Reply:
Az AlphaVantage API-ról jövő adatokat megszűntetem teljesen, a Stock2-ből csináltam Stock-ot, a függvény neveket is javítom, hogy ne legyen szám a nevében.